perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_resource_attribute_def_virt_isBanned.java

```diff
@@ -109,9 +108,9 @@ public Attribute getAttributeValue(PerunSessionImpl sess, Member member, Resourc
 		try {
 			Member member = perunSession.getPerunBl().getMembersManagerBl().getMemberById(perunSession, banSetForResource.getMemberId());
 			Resource resource = perunSession.getPerunBl().getResourcesManagerBl().getResourceById(perunSession, banSetForResource.getResourceId());
-			Attribute attribute = perunSession.getPerunBl().getAttributesManagerBl().getAttribute(perunSession, member, resource, A_MR_V_isBanned);
-			resolvingMessages.add(new AttributeSetForResourceAndMember(attribute, resource, member));
-		} catch (MemberNotExistsException | ResourceNotExistsException | MemberResourceMismatchException e) {
+			AttributeDefinition attributeDefinition = perunSession.getPerunBl().getAttributesManagerBl().getAttributeDefinition(perunSession, A_MR_V_isBanned);
+			resolvingMessages.add(new AttributeSetForResourceAndMember(new Attribute(attributeDefinition), resource, member));
```

Comment:
I'm not sure, we should keep using ``AttributeSetForResourceAndMember`` message, while we otherwise use ``AttributeChangedFor*`` class. Same goes for "remove" case.

Reply:
Okay, the messages were changed to use the new type of audit event.